### ISSUE TITLE
Missing glyphs in Wine

### ISSUE THREAD

**Issue by dangbunni** [BeipDev/BeipMU#179]
Running BeipMU in Wine is missing glyphs. It's almost as if a font is missing, but installing all fonts doesn't help.

I get this: https://i.imgur.com/DWiRofm.png

I get this error message: https://i.imgur.com/hX2WG2V.png

I tried overriding the d2d1 DLL with the Windows native version and that suppresses the error, but the whole thing crashes immediately.

Manjaro XFCE, current. A friend tried on current Mint and got the same issue.

**Comment by BeipDev** [comment]
Ah, I've seen that myself. The font issue is just that I needed to install "Segoe UI Emoji" (named seguiemj.ttf on my Windows install). But the Direct2D Wine error happened to me whenever I ran my Linux VM with hardware acceleration. I figured that was a VirtualBox issue, not a Wine one.

This is on wine-8.11 (Staging)

<img width="524" alt="image" src="https://github.com/BeipDev/BeipMU/assets/30326317/fc68eabf-a4bb-42c2-95d3-84ed623e670f">
